### PR TITLE
fixed some grammar mistakes

### PR DESCRIPTION
**Pull request by AmirNaghibi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ http://pcottle.github.com/learnGitBranching/
 By default the application launches in "sandbox mode" with a basic repository already created. Here you can enter commands and mess around with a repository as much as you like. Keep in mind you can
 
 * `undo` to undo the effects of the last command
-* `reset` to start over from a clean slate (works in levels too)
+* `reset` to start over with a clean slate (works in levels too)
```
Comment (waldyrious) [inline README.md]: I don't agree that this line needs changing. "Start from" (and by extension "start over from") is perfectly valid English AFAIK.

File: README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ http://pcottle.github.io/learnGitBranching/?gist_level_id=a84407351f9c9f0cb241
 
 ## Reporting Bugs / Opening Issues
 
-When reporting bugs, try running the command `debug_copyTree()` in your JS console when in a state just before reproducing a bug. This can avoid having to copy over all the commands you used to get into a specific state. (I can then use the `importTreeNow` command to get to that exact state)
+When reporting bugs, try running the command `debug_copyTree()` in your JS console when in a state just before reproducing a bug. This can avoid having to copy all the commands you used to get into a specific state. (I can then use the `importTreeNow` command to get to that exact state)
```
Comment (waldyrious) [inline README.md]: There's nothing wrong with "copy over" either. I see it as a similar construction to "move over".

**Comment by waldyrious** [comment]
Nice changes overall :+1:, but I disagree with two of the changes, as indicated in the inline comments.

**Comment by pcottle** [comment]
Restarted the build and it passed! so thats good.

I agree that some of these are borderline but I really appreciate the pull request!! So going to merge based on that, code wins arguments :) 🎉

**Comment by waldyrious** [comment]
Good call. Welcoming contributions is certainly more important than pedantry. Besides, the new phrasings aren't incorrect either :)

**Pull request merged**